src/status_im2/core_spec.cljs

```diff
@@ -1,4 +1,5 @@
 (ns status-im2.core-spec
   (:require
-    [status-im2.contexts.communities.actions.community-options.component-spec]
-    [status-im2.contexts.chat.messages.content.audio.component-spec]))
+   ;    [status-im2.contexts.communities.actions.community-options.component-spec]
```

Comment:
?

Reply:
I will check again, but last time these component tests were broken and I had commented them temporarily